packages/react-components/react-table/library/src/components/DataGridRow/DataGridRow.test.tsx

```diff
@@ -183,20 +183,6 @@ describe('DataGridRow', () => {
       expect(toggleRow).toHaveBeenCalledTimes(0);
     });
 
-    it('should disable selection cell input if the row is in a header and the selection mode is singe', () => {
-      const toggleRow = jest.fn();
-      const ctx = mockDataGridContext({ selectableRows: true }, { selection: { toggleRow, selectionMode: 'single' } });
-      const { container } = render(
-        <DataGridHeader>
-          <DataGridContextProvider value={ctx}>
-            <DataGridRow>{() => <div />}</DataGridRow>
-          </DataGridContextProvider>
-        </DataGridHeader>,
-      );
-
-      expect(container.querySelector("input[type='radio']")?.hasAttribute('disabled')).toBe(true);
-    });
-
```

Comment:
I removed this rather than adding a new one b/c the only change now is in the styles